xml/templates/dao.tpl

```diff
@@ -72,7 +72,11 @@ class {$table.className} extends CRM_Core_DAO {ldelim}
      * {$field.comment|regex_replace:"/\n[ ]*/":"\n* "}
      *
 {/if}
-     * @var {$field.phpType}
+     * @var {$field.phpType}{if $field.phpNullable}|null
+{/if}
+
```

Comment:
I had some issues with smarty ignoring my newline after the closing {/if}, hence the slightly odd formatting here.